language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
+COM_USERS_REGISTER_USERNAME_LABEL="Nazwa użytkownika"
+COM_USERS_REGISTER_USERNAME_MESSAGE="Podana nazwa użytkownika jest niepoprawna lub jest już użyta. Wpisz inną nazwę użytkownika."
 COM_USERS_REGISTRATION="Rejestracja użytkownika"
 COM_USERS_REGISTRATION_ACL_ADMIN_ACTIVATION="Zaloguj się, aby potwierdzić, że masz uprawnienia do aktywowania nowych kont."
-COM_USERS_REGISTRATION_ACL_ADMIN_ACTIVATION_PERMISSIONS="Nie masz upoważnienia do aktywowania nowych kont, zaloguj się za pomocą konta uprzywilejowanego."
-COM_USERS_REGISTRATION_ACTIVATE_SUCCESS="Dziękujemy za rejestrację. Twoje konto zostało uaktywnione. Możesz się teraz logować, korzystając z  nazwy użytkownika i hasła podanych podczas rejestracji."
+COM_USERS_REGISTRATION_ACL_ADMIN_ACTIVATION_PERMISSIONS="Nie masz uprawnień do aktywowania nowych kont, zaloguj się na konto uprzywilejowane."
+COM_USERS_REGISTRATION_ACTIVATE_SUCCESS="Twoje konto zostało uaktywnione. Możesz teraz zalogować się, korzystając z nazwy użytkownika i hasła podanych podczas rejestracji."
 COM_USERS_REGISTRATION_ACTIVATION_NOTIFY_SEND_MAIL_FAILED="Wystąpił błąd podczas wysyłania wiadomości e-mail potwierdzającej aktywację konta"
 COM_USERS_REGISTRATION_ACTIVATION_SAVE_FAILED="Błąd podczas zapisywania danych aktywacyjnych: %s"
-COM_USERS_REGISTRATION_ADMINACTIVATE_SUCCESS="Konto użytkownika zostało pomyślnie zatwierdzone, a użytkownik został o tym poinformowany."
-COM_USERS_REGISTRATION_COMPLETE_ACTIVATE="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na wprowadzony adres e-mail. Kolejnym krokiem jest konieczność aktywacji konta, uczynisz to, klikając odnośnik aktywacyjny w wiadomości e-mail, którą dostaniesz. Aktywacja konta jest wymagana przed pierwszym logowaniem na stronie."
-COM_USERS_REGISTRATION_COMPLETE_VERIFY="Twoje konto zostało utworzone i odnośnik weryfikacyjny został wysłany na wprowadzony adres e-mail. Kolejnym krokiem jest konieczność weryfikacji konta, uczynisz to, klikając odnośnik weryfikacyjny w wiadomości e-mail, którą dostaniesz. Po pomyślnej weryfikacji Administrator zatwierdzi Twoje konto przed  pierwszym logowaniem."
+COM_USERS_REGISTRATION_ADMINACTIVATE_SUCCESS="Konto użytkownika zostało pomyślnie aktywowane i użytkownik został o tym powiadomiony."
+COM_USERS_REGISTRATION_COMPLETE_ACTIVATE="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na podany adres e-mail. Pamiętaj, że musisz aktywować konto, wybierając odnośnik aktywacyjny w otrzymanej wiadomości e-mail, którą dostaniesz. Aktywacja konta jest wymagana przed pierwszym zalogowaniem się na stronie internetowej."
+COM_USERS_REGISTRATION_COMPLETE_VERIFY="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na podany adres e-mail. Pamiętaj, że musisz aktywować konto, wybierając odnośnik aktywacyjny w otrzymanej wiadomości e-mail, którą dostaniesz. Aktywacja konta i weryfikacja przez administratora jest wymagane przed pierwszym zalogowaniem się na stronie internetowej."
```

Komentarz w wierszu 116
```suggestion
COM_USERS_REGISTRATION_COMPLETE_VERIFY="Twoje konto zostało utworzone i odnośnik aktywacyjny został wysłany na podany adres e-mail. Pamiętaj, że musisz aktywować konto, wybierając odnośnik aktywacyjny w otrzymanej wiadomości e-mail, którą dostaniesz. Aktywacja konta i weryfikacja przez administratora jest wymagane przed pierwszym zalogowaniem się do witryny."
```